src/integration/Tests.cs

```diff
@@ -180,6 +182,37 @@ private static void TestReturnsCorrectNumberOfResultsViaProxy(ICredentials crede
 			AssertResults("VIA_PROXY", citiesAmount, 7);
 		}
 
+		private static void TestUsEnrichmentPropertyPrincipalRequestReturnsCorrectNumberOfResults(ICredentials credentials){
+			var client = new ClientBuilder(credentials).BuildUsEnrichmentApiClient();
+			
+			SmartyStreets.USEnrichmentApi.Property.Principal.Result[] results = null;
+            try
+            {
+                results = client.SendPropertyPrincipalLookup("1682393594");
+            }
+            catch (Exception)
+            {
+                Console.WriteLine("");
+            }
+			AssertResults("ENRICHMENT_PROPERTY_PRINCIPAL", results.Length, 1);
+		}
+
+		private static void TestUsEnrichmentPropertyFinancialRequestReturnsCorrectNumberOfResults(ICredentials credentials){
+			var client = new ClientBuilder(credentials).BuildUsEnrichmentApiClient();
+			
+			SmartyStreets.USEnrichmentApi.Property.Financial.Result[] results = null;
+            try
+            {
+                results = client.SendPropertyFinancialLookup("1682393594");
+            }
+            catch (Exception)
+            {
+                Console.WriteLine("");
```

Comment:
Was this meant to be left blank?

Reply:
Yes, I am just following the pattern set in other tests. A little odd I agree.